src/main/scala/outwatch/dom/package.scala

```diff
@@ -23,4 +25,9 @@ package object dom extends Implicits with ManagedSubscriptions {
 
   type Handler[T] = outwatch.Handler[T]
   val Handler = outwatch.Handler
+
+
+  def sideEffect[T](f: T => Unit)(implicit s: Scheduler): Sink[T] = Sink.create[T] { e => IO { f(e); Continue } }(s)
+  def sideEffect[S,T](f: (S,T) => Unit)(implicit s: Scheduler): Sink[(S,T)] = Sink.create[(S,T)] { e => IO { f(e._1, e._2); Continue } }(s)
+  def sideEffect(f: => Unit)(implicit s: Scheduler): Sink[Any] = Sink.create[Any] { e => IO { f; Continue } }(s)
```

Comment:
It would be nicer if these were in a `SideEffects` trait rather than in the package object.